pylint_nautobot/model_label.py

```diff
@@ -0,0 +1,89 @@
+"""Check model label construction in f-strings."""
+from astroid import Attribute
+from astroid import Const
+from astroid import FormattedValue
+from astroid import JoinedStr
+from astroid import Name
+from astroid import NodeNG
+from pylint.checkers import BaseChecker
+from pylint.interfaces import HIGH
+from pylint.interfaces import IAstroidChecker
+
+
+def _get_model_name(node: NodeNG, attrname: str) -> str:
```

Comment:
Function name is not very clear as to its meaning/usage. Maybe something like `_check_for_model_meta_attr` or something might be clearer? Also please add a docstring explaining what this returns. I *think* it returns either an empty string or the name of a "model" variable, but I'm not sure I'm reading this correctly.

Reply:
Should be better named and documented now.